File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/RetailIdCreateRetailIdAssociateRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class RetailIdCreateRetailIdAssociateRequestItem {
    @JsonProperty("PackageLabel")
    public String packageLabel;
    @JsonProperty("QrUrls")
    public List<String> qrUrls;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/RetailIdCreateRetailIdAssociateRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class RetailIdCreateRetailIdAssociateRequestItem(
    @JsonProperty("packageLabel")
    val packageLabel: String? = null,
    @JsonProperty("qrUrls")
    val qrUrls: String? = null
)
